tests/foreman/cli/test_remoteexecution.py

```diff
@@ -583,6 +584,68 @@ def test_positive_run_reccuring_job(self, fixture_vmsetup, module_org):
         assert rec_logic['state'] == 'finished'
         assert rec_logic['iteration'] == '2'
 
+    @pytest.mark.tier3
+    @pytest.mark.parametrize('fixture_vmsetup', ['rhel7'], indirect=True)
+    def test_positive_run_concurrent_jobs(self, fixture_vmsetup, module_org):
+        """Tests Ansible REX concurent jobs without batch trigger
+
+        :id: ad0f108c-03f2-49c7-8732-b1056570567b
+
+        :Steps:
+
+            0. Create 2 hosts, disable foreman_tasks_proxy_batch_trigger
+
+            1. Run Ansible Command job with concurrency-setting
+
+        :expectedresults: multiple asserts along the code
+
+        :CaseAutomation: Automated
+
+        :CaseLevel: System
+
+        :BZ: 1817320
+
+        :parametrized: yes
+        """
+        param_name = 'foreman_tasks_proxy_batch_trigger'
+        GlobalParameter().set({'name': param_name, 'value': 'false'})
+        self.client = fixture_vmsetup
+        output_msgs = []
+        with VMBroker(nick=fixture_vmsetup, host_classes={'host': ContentHost}) as client2:
```

Comment:
Since `fixture_vmsetup` is a method that is indirectly parametrized, I believe it will be the client instance here, not  just the indirectly parametrized string.

Reply:
fixture_vmsetup.param is what I had in mind, cheers